src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -451,12 +451,347 @@ The following changes must be performed:
+
+The <<../user/ProActiveUserGuide.adoc#_generic_information,*Generic Information*>> `NS` allows a ProActive Task to be executed on a Native Scheduler node source.
+
+It must contain the name of the target node source. Example:
+
+```xml
+<task name="PBS_Task">
+    <description>
+        <![CDATA[ Execute this Task in the PBS node source. ]]>
+    </description>
+    <genericInformation>
+        <info name="NS" value="PBS"/>
+    </genericInformation>
+    <scriptExecutable>
+        <script>
+          <code language="groovy">
+            <![CDATA[
+println "Hello World"
+]]>
+          </code>
+        </script>
+    </scriptExecutable>
+</task>
+```
+
+The NS value can also be defined at the job level, in that case, every tasks of this job will be executed on the Native Scheduler node source:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.10"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.10 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.10/schedulerjob.xsd"
+    name="PBS_Job"
+    priority="normal"
+    onTaskError="continueJobExecution"
+     maxNumberOfExecution="2"
+>
+  <genericInformation>
+    <info name="NS" value="PBS"/>
+  </genericInformation>
+  <taskFlow>
+    <task name="Groovy_Task">
+      <description>
+        <![CDATA[ The simplest task, ran by a groovy engine. ]]>
+      </description>
+      <scriptExecutable>
+        <script>
+          <code language="groovy">
+            <![CDATA[
+println "Hello World"
+]]>
+          </code>
+        </script>
+      </scriptExecutable>
+    </task>
+  </taskFlow>
+</job>
+```
+
+It is also possible to define in a ProActive Task or Job the `#BATCH` generic information. This parameter allows to provide custom arguments to `nsSubmitCommand`.
+
+For example:
+
+```xml
+<task name="PBS_Task">
+  <description>
+    <![CDATA[ Runs on the PBS node source on queue1, using 2 nodes * 2 cpus ]]>
+  </description>
+  <genericInformation>
+    <info name="NS" value="PBS"/>
+    <info name="#BATCH" value="-q queue1 -lnodes=2:ppn=2"/>
+  </genericInformation>
+  <scriptExecutable>
+    <script>
+      <code language="groovy">
+         <![CDATA[
+println "Hello World"
+]]>
+      </code>
+    </script>
+  </scriptExecutable>
+</task>
+```
+
+==== Native Scheduler Node Life Cycle
+
+As soon as tasks containing the `NS` generic information are pending, the target Native Scheduler node source will try to deploy ProActive Nodes to execute them.
+
+The node will first appear in `Deploying` <<_node_states,*state*>>. If some error occurs prior to the `nsSubmitCommand` execution (SSH connection, command syntax), the node state will change to `Lost`, with some explanation about the failure displayed.
+
+If the node remains in `Deploying` state, it is possible to monitor the job execution on the native scheduler itself (by logging into the head node):
```

Comment:
We should note here that doing that is not a ProActive feature. They are scheduler specific commands.

Reply:
right